### PR TITLE
refactor(ecs): change get service arn method to a public func in ecs pkg

### PR DESCRIPTION
**Pull request by iamhopaul123**
Refactor get service arn method in `describe` pkg to a public func in `ecs` pkg for reusability.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by huanjani — APPROVED** [review]
looks good to me!

File: internal/pkg/describe/status.go [hunk 1]
```diff
@@ -14,14 +14,13 @@ import (
 	"github.com/aws/copilot-cli/internal/pkg/aws/aas"
 	"github.com/aws/copilot-cli/internal/pkg/aws/cloudwatch"
 	"github.com/aws/copilot-cli/internal/pkg/aws/ecs"
-	rg "github.com/aws/copilot-cli/internal/pkg/aws/resourcegroups"
 	"github.com/aws/copilot-cli/internal/pkg/aws/sessions"
 	"github.com/aws/copilot-cli/internal/pkg/deploy"
+	copiEcs "github.com/aws/copilot-cli/internal/pkg/ecs"
```
Comment (efekarakus) [inline internal/pkg/describe/status.go]: nit: I'd instead rename the `aws/ecs` to `awsECS`. Reading "copiEcs" is kinda odd :P

File: internal/pkg/ecs/ecs.go [hunk 2]
```diff
@@ -62,9 +63,29 @@ func (c Client) Cluster(app, env string) (string, error) {
 	return clusters[0].ARN, nil
 }
 
+// ServiceARN returns the ARN of the ECS service for a Copilot service.
```
Comment (efekarakus) [inline internal/pkg/ecs/ecs.go]:
```suggestion
// ServiceARN returns the ARN of an ECS service created with Copilot.
```

File: internal/pkg/ecs/ecs.go [hunk 3]
```diff
@@ -62,9 +63,29 @@ func (c Client) Cluster(app, env string) (string, error) {
...
+func (c Client) ServiceARN(app, env, svc string) (*ecs.ServiceArn, error) {
+	services, err := c.rgGetter.GetResourcesByTags(serviceResourceType, map[string]string{
+		deploy.AppTagKey:     app,
+		deploy.EnvTagKey:     env,
+		deploy.ServiceTagKey: svc,
+	})
+	if err != nil {
+		return nil, fmt.Errorf("get ECS service for %s in environment %s: %w", svc, env, err)
```
Comment (efekarakus) [inline internal/pkg/ecs/ecs.go]: nit
```suggestion
		return nil, fmt.Errorf("get ECS service with tags (%s, %s, %s): %w", app, env, svc, err)
```

File: internal/pkg/ecs/ecs.go [hunk 4]
```diff
@@ -62,9 +63,29 @@ func (c Client) Cluster(app, env string) (string, error) {
...
+	}
+	if len(services) == 0 {
+		return nil, fmt.Errorf("no ECS service found for %s in environment %s", svc, env)
+	}
+	if len(services) > 1 {
+		return nil, fmt.Errorf("more than one ECS service is found for %s in environment %s", svc, env)
```
Comment (efekarakus) [inline internal/pkg/ecs/ecs.go]: nit:
```suggestion
		return nil, fmt.Errorf("more than one ECS service with the name %s found in environment %s", svc, env)
```

**Review by efekarakus — COMMENTED** [review]
Looks good! just tiny error message improvements for users

**Review by efekarakus — APPROVED** [review]

**Pull request merged**